.nvmrc

```diff
@@ -0,0 +1 @@
+v10.14.0
```

Comment:
```suggestion
v11.2.0
```

:wink:

Reply:
i don't agree, i see no reason to not use an lts version in prod unless there's something new and fancy we really want.

Reply:
Agreed!